### ISSUE TITLE
Dynamic loading/ordering of facet values

### ISSUE THREAD

**Issue by allanaaa** [omeka-s-modules/FacetedBrowse#25]
How hard would it be for Faceted Browse facets to dynamically update whenever the page is loaded? 

Rather than a bunch of text entries loaded from existing values at one time, I would like the top facets to display the values with the highest counts at the time the user visits the public page.

Currently you can load facet values on the edit page and they display with the current count - but then when you add them they get frozen in that order forever. Presumably users will want to make their facets and then continue adding items and editing values and not have to update the browse pages every so often.

![facetedbrowse2](https://user-images.githubusercontent.com/28012148/202018212-83f84452-3adf-4a73-803b-67cce84d32b5.PNG)

I generated the values list (top of the image) earlier, then added another import, so the counts now stand at what you can see in the lower part. 

Ideally the user on the public page would also be able to sort any browse facet in the sidebar by either alphabetical / chronological or by the count of resources.

**Comment by jimsafley** [comment]
Yes, I agree that the features you're asking for are desirable, and perhaps expected, but they're just not possible. It's important to note that this module is a a thin layer on top of our existing advanced search, not a fully-featured enterprise search engine. If users need the features your describe, they can use the [Solr module](https://omeka.org/s/modules/Solr/).
